Import/SampleForLabelsInReports/Import/Report/Part Report With Label.xml

```diff
@@ -9,100 +9,101 @@
+	  <xsl:template match="/">
+		    <html>
+			      <head/>
+			      <style type="text/css" userData="Global">
+				.cellSolid {border-top:1px #000000 solid;border-right:1px #000000 solid;border-bottom:1px #000000 solid;border-left:1px #000000 solid;}
+				.cellSolidTopRight {border-top:1px #000000 solid;border-right:1px #000000 solid;}
+				.cellSolidTopLeft {border-top:1px #000000 solid;border-left:1px #000000 solid;}
+				.cellSolidRightBottom {border-right:1px #000000 solid;border-bottom:1px #000000 solid;}
+				.cellSolidBottomLeft {border-bottom:1px #000000 solid;border-left:1px #000000 solid;}
+				.cellSolidTop {border-top:1px #000000 solid;}
+				.cellSolidRight {border-right:1px #000000 solid;}
+				.cellSolidBottom {border-bottom:1px #000000 solid;}
+				.cellSolidLeft {border-left:1px #000000 solid;}
+				.cellDashed {border-top:1px #666666 dashed;border-right:1px #666666 dashed;border-bottom:1px #666666 dashed;border-left:1px #666666 dashed;}
+				.cellDashedTopRight {border-top:1px #666666 dashed;border-right:1px #666666 dashed;}
+				.cellDashedTopLeft {border-top:1px #666666 dashed;border-left:1px #666666 dashed;}
+				.cellDashedBottomRight {border-bottom:1px #666666 dashed;border-right:1px #666666 dashed;}
+				.cellDashedBottomLeft {border-bottom:1px #666666 dashed;border-left:1px #666666 dashed;}
+				.cellDashedTop {border-top:1px #666666 dashed;}
+				.cellDashedRight {border-right:1px #666666 dashed;}
+				.cellDashedBottom {border-bottom:1px #666666 dashed;}
+				.cellDashedLeft {border-left:1px #666666 dashed;}
+				.cellHeader {background-color:#CCCCCC;border-top:1px #000000 solid;border-right:1px #000000 solid;border-bottom:1px #000000 solid;padding:2px;text-align:center;text-transform:capitalize;text-align:center;font-family:helvetica;font-weight:bold;font-size:8pt;}
+			</style>
+			      <body topmargin="50" leftmargin="50">
+				        <table border="0" cellspacing="0" cellpadding="0" width="950">
+					          <tr>
+						            <td width="150" uniqueID="ms__id2"/>
+						            <td width="200" uniqueID="ms__id3"/>
+						            <td width="150" uniqueID="ms__id4"/>
+						            <td width="150" uniqueID="ms__id5"/>
+						            <td width="150" uniqueID="ms__id6"/>
+					  </tr>
+					          <tr valign="top">
+						            <td align="left" uniqueID="ms__id77" colspan="9">
+							              <img src="../images/aras-innovator.svg" height="40"/>
+						      </td>
+					  </tr>
+					          <tr valign="bottom">
+						            <td colspan="2" style="font-family:helvetica;font-size:15pt;color:#DA1943;padding:2px;" align="left" uniqueID="ms__id79">Part Report (with Make/Buy Labels)</td>
+						            <td colspan="3" style="font-family:helvetica;font-size:10pt;padding:2px;" align="right" uniqueID="ms__id80">
+							            <xsl:value-of select="//Item/timestamp"/>
+							                <xsl:if test="timestamp='' or not(timestamp)">
```

Comment:
I don't think any of these if statements are necessary. If the property is empty, the value-of tag should just output nothing.